examples/mnist/mnist_deep_simplified.py

```diff
@@ -172,7 +172,7 @@ def train_mnist_cnn(FLAGS):
         if FLAGS.optimizer == "adam":
             train_step = tf.train.AdamOptimizer(1e-4).minimize(cross_entropy)
         elif FLAGS.optimizer == "sgd":
-            train_step = tf.train.GradientDescentOptimizer(1e-4).minimize(
+            train_step = tf.train.GradientDescentOptimizer(5e-2).minimize(
```

Comment:
earlier with sgd, we were converging to 20%. Increased LR to converge to >95% now. Just so that the sample script provides some reasonable mnist numbers.